services/app-api/package.json

```diff
@@ -19,4 +19,4 @@
   "dependencies": {
     "uuid": "^7.0.3"
   }
-}
+}
```

Comment:
Add a newline at the end of 22 so it matches what is in master and it will take this file out of one that's changed from master please